src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import duke.exception.DescriptionEmptyException;
+import duke.exception.DukeException;
+import duke.exception.IncorrectFormatException;
+import duke.exception.InvalidTaskTypeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class Parser {
+
+    private TaskList taskList;
```

Comment:
Would tasks be a better name? Since plural form should be used on names representing a collection of objects 🤔